dist/22-5490-schema.json

```diff
@@ -25,7 +25,7 @@
         "testReimbursement",
         "licensingReimbursement",
         "tuitionTopUp",
-        "farmCoop"
+        "cooperativeTraining"
```

Comment:
we should add cooperativeTraining and leave farmCoop in the backend/schema for a week to avoid errors, deploy the changes to frontend first and then remove farmCoop from the backend schema later